BDArmory/UI/LoadedVesselSwitcher.cs

```diff
@@ -337,7 +337,7 @@ private void WindowVesselSwitcher(int id)
             int numButtons = 10;
             int numButtonsOnLeft = 5;
             GUI.DragWindow(new Rect(numButtonsOnLeft * _buttonHeight + _margin, 0f, BDArmorySettings.VESSEL_SWITCHER_WINDOW_WIDTH - numButtons * _buttonHeight - 3f * _margin, _titleHeight));
-
+            //GUI.Label(new Rect(5f * _buttonHeight + _margin * 2, 4f, _buttonHeight * 10, _buttonHeight), BDArmorySetup.Instance.Version);
```

Comment:
Could enable this if we want a version watermark on the VS vs only during comps

Reply:
It'd probably look better on the right hand side, e.g.,
```csharp
GUI.Label(new Rect(BDArmorySettings.VESSEL_SWITCHER_WINDOW_WIDTH - (numButtons - numButtonsOnLeft) * _buttonHeight - _margin - 70f, 4f, 70f, _titleHeight - 4f), BDArmorySetup.Instance.Version);
```
![Screenshot_20221123_134522](https://user-images.githubusercontent.com/18440126/203550295-62dfec14-5e80-43b4-b5ae-1fa93a09acaf.png)
but otherwise, yeah, it wouldn't hurt to have that. It'd still only be visible when the UI is hidden if the option to always show the vessel switcher is enabled though, so we'd probably still want the other version watermark.

Also, changing the `_windowHeight` line at the end of the function to
```csharp
_windowHeight = Mathf.Max(height, _titleHeight + _buttonHeight);
```
avoids having the title text cut in half.